ui/components/institutional/note-to-trader/note-to-trader.stories.tsx

```diff
@@ -9,13 +9,13 @@ export default {
       'The approver will see this note when approving the transaction at the custodian.',
     noteText: '',
     labelText: 'Transaction note',
-    maxLength: '280',
+    maxLength: 280,
     onChange: () => {
       /**/
     },
   },
 };
 
-export const DefaultStory = (args) => <NoteToTrader {...args} />;
+export const DefaultStory = (args: any) => <NoteToTrader {...args} />;
```

Comment:
In this case we can define a type instead of any from observing the `args` values above

Reply:
@pedronfigueiredo changes are done